backend/config.py

```diff
@@ -1,58 +1,29 @@
-from sqlalchemy import create_engine, Column, Integer, DateTime, JSON, func
-from contextlib import contextmanager
-
-
-import os
-import sys
+import json
 import logging
-import importlib.metadata
-import pkgutil
-from urllib.parse import urlparse
+import os
+import shutil
 from datetime import datetime
-
-import chromadb
-from chromadb import Settings
-from typing import TypeVar, Generic
-from pydantic import BaseModel
-from typing import Optional
-
 from pathlib import Path
-import json
-import yaml
+from typing import Generic, Optional, TypeVar
+from urllib.parse import urlparse
 
+import chromadb
 import requests
-import shutil
-
-
+import yaml
 from apps.webui.internal.db import Base, get_db
-
-from constants import ERROR_MESSAGES
-
 from env import (
```

Comment:
Deleting these from config will break the code, revert needed here!

Reply:
Why do we have to keep this variable as it not accessed?

Reply:
I think I have the answer with this PR: https://github.com/open-webui/open-webui/pull/4957 !
We can delete it we just need to be sure the imports point to the correct files everytime 🙂 

We can think to implement the ruff formatting by default to avoid these kind of issues in the future